client/src/components/HostViews/IntroView.vue

```diff
@@ -19,7 +23,47 @@ export default {
   mixins: [
     HostMixin
   ],
+  data() {
+    return {
+      players: [],
+      votesForSkip: 0
+    }
+  },
+  mounted() {
+    this.playerList.forEach(player => {
+      this.players.push({
+        playerName: player,
+        wantsToSkip: false
+      })
+    })
+    this.socketInstance.on('skip-vote', (vote) => {
+      for (let i = 0; i < this.players.length; i++) {
+        if (this.players[i].playerName === vote.playerName) {
+          this.players[i].wantsToSkip = vote.wantsToSkip
+          this.reComputeSkipCount()
+          break
+        }
+      }
+    })
+  },
+  computed: {
+    votesAgainstSkip() {
+      return this.players.length - this.votesForSkip
+    }
+  },
   methods: {
+    reComputeSkipCount() {
+      let votesForSkip = 0
+      this.players.forEach(player => {
+        if (player.wantsToSkip) votesForSkip++
+      })
+      this.votesForSkip = votesForSkip
+    
+      // check if everyone is on board to skip
+      if (this.players.every(player => player.wantsToSkip)) {
```

Comment:
Can we not assume if `this.players.length == this.votesForSkip` is true that every player would want to skip?

Reply:
It is for the edge case that no players are added to the list before that runs. Doing it like this ensures that I dont have to use a bunch of chained logic in the "if" statement for the behavior that I want, which is that it only triggers if every player has voted, and that there specifically has to be at least one player who wants to skip and actually voted

Reply:
if this makes the conditional unnecessarily hard to read, we can do what you wrote, because it is perfectly valid

Reply:
This is fine for a sanity check. I might run both though to just confirm if you're looking for a sanity check though.

Reply:
Thank you offering the advice, made appropriate adjustment